source/community.html.erb

```diff
@@ -33,8 +33,18 @@ responsive: true
   <p>
     Please note that neither the IRC channel nor the Slack room are managed or moderated by the Ember Core Team. They are public forums.
   </p>
+</div>
+
+<div class="community section newsletters">
+  <h2>Stay Up to Date with the Latest News</h2>
+  <p>
+    <a href="https://the-emberjs-times.ongoodbits.com/">Ember.js Times</a> &mdash; follow the progress of Ember feature development, requests for community input (RFCs), and calls for contributors
```

Comment:
I wonder if it might make sense to widen the term `Ember feature development` a bit, since we also write about projects outside of Ember, e.g. Glimmer, website things or community addons. Maybe something like  `contributions to the Ember ecosystem` might be useful to be added. 
What are your thoughts?